docs/analysis/criteria.md

```diff
@@ -174,48 +206,31 @@ Alternatively, files should be brought into the website repo via
 If a project chooses to keep source files in multiple repos, they need a clearly
 documented strategy for managing mirrored files and new contributions.
 
+We evaluate on the following:
+
+- Does the project have a single source for documentation? If not, is there a
+  reason?
+
 ### Minimal website requirements
 
 Listed here are the _minimal_ website requirements for projects based on their
-[maturity level][]: sandbox, incubating, graduated and archived.
+[maturity level]: sandbox, incubating, graduated and archived.
```

Comment:
Keep the trailing `[]`, it helps tools like VS Code know that this is, for sure, a markdown link reference:

```suggestion
[maturity level][]: sandbox, incubating, graduated and archived.
```